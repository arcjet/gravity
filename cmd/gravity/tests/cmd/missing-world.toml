bin.name = "gravity"
args = "--world nonexistent ../../target/wasm32-unknown-unknown/release/example_basic.wasm"
status.code = 1
