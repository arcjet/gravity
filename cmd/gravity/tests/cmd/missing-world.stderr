unable to find world: nonexistent
